doc/designs/ddd/ddd-03-fd-blocking.c

```diff
@@ -190,7 +190,8 @@ int main(int argc, char **argv)
         goto fail;
     }
 
-    if (tx(ssl, msg, sizeof(msg)-1) < sizeof(msg)-1) {
+    l = tx(ssl, msg, mlen);
+    if (l < mlen) {
```

Comment:
Same as before, I find this less instantly readable.

Reply:
As before, if you look a few lines down, then you see that pattern was already used with `rx()`.

Reply:
Pauli, is it okay to mark this as resolved?